src/plan_bot.py

```diff
@@ -76,18 +76,15 @@ def process_post(
             plan_confidence = match_info["confidence"]
             plan = match_info["plan"]
             plan_id = plan["id"]
-            post_parent_id = False # Could be "" unless constructing URLs with it later
-            post_title = False # Could be "" unless constructing URLs with it later
-            post_top_level_parent_id = False
-
-            if(self.type == "comment"):
-                # Reddit 3-digit code prefix removed for each id, leaving only the ID itself
-                post_parent_id = post.parent_id([3:])
```

Comment:
`post.parent_id([3:])` was a python syntax error, but I also moved it into a ternary to make variable definition more atomic